src/datachain/client/gcs.py

```diff
@@ -32,6 +32,14 @@ def create_fs(cls, **kwargs) -> GCSFileSystem:
 
         return cast(GCSFileSystem, super().create_fs(**kwargs))
 
+    def url(self, path: str, expires: int = 3600, **kwargs) -> str:
```

Comment:
What's calling this function?

Reply:
Entrypoint I came from is SaaS calling this function: https://github.com/iterative/datachain/blob/main/src/datachain/catalog/catalog.py#L1199-L1202

Reply:
Oh, I see! 🙏